src/cheri/hypervisor-integration.adoc

```diff
@@ -189,5 +188,7 @@ load and store capability instructions behave as described in
 xref:section_existing_riscv_insns[xrefstyle=short]. In
 {cheri_int_mode_name}, the instructions use the low XLEN bits of the base register as the effective address for the memory access and the capability authorizing the memory access is <<ddc>>.
 
+ifdef::cheri_standalone_spec[]
```

Comment:
Isn't this always true now that you are no longer including this file?

Reply:
yes - tags removed